### PR TITLE
chore: Add missing entries to CHANGELOG.md

### PR DESCRIPTION
**Pull request by aslafy-z**
Add missing entries to CHANGELOG.md

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,9 +3,12 @@
 
 All notable changes to this project will be documented here.
 
-### v2.2.0
+### v2.1.11
 - Feature: Add topologySpreadConstraints [PR-239](https://github.com/stakater/application/pull/239)
 
+### v2.1.10
+- Chore: Add some basic unit tests and update CI workflow [PR-218](https://github.com/stakater/application/pull/239)
```
Comment (aslafy-z) [inline CHANGELOG.md:10]:
```suggestion
- Chore: Add some basic unit tests and update CI workflow [PR-218](https://github.com/stakater/application/pull/218)
```

**Comment by mustafaStakater** [comment]
Ah, i need to look into automatic changelog updates

also , this will cause main pipeline to be triggered because `skip-ci` might not work according to ref below

If any commit message in your push or the HEAD commit of your PR contains the strings [skip ci], [ci skip], [no ci], [skip actions], or [actions skip] workflows triggered on the push or pull_request events will be skipped.
Ref: https://github.blog/changelog/2021-02-08-github-actions-skip-pull-request-and-push-workflows-with-skip-ci/

**Comment by aslafy-z** [comment]
@mustafaStakater This is the job I want to skip: https://github.com/stakater/application/blob/master/.github/workflows/push.yaml#L21. Will this pipeline still triggered?

**Comment by mustafaStakater** [comment]
yes pipeline will be triggered, for some reason, `[skip-ci]` in commit message doesnt skip workflows, i wasnt able to investigate it.

**Comment by aslafy-z** [comment]
Ok. I just updated the commit with 'skip ci' instead of 'skip-ci'

**Review by mustafaStakater — APPROVED** [review]

**Comment by aslafy-z** [comment]
@mustafaStakater it did not work. The release job was executed :(

**Pull request merged**